lib/bank/money_market_account.rb

```diff
@@ -0,0 +1,69 @@
+module Bank
+  class MoneyMarketAccount < Account
+    attr_accessor :num_of_transactions, :balance, :interest, :month
+
+    def initialize(id, initial_balance)
+      raise ArgumentError, "Account requires minimum balance of $10,000." if initial_balance.to_i < 10_000
+      super
+      @num_of_transactions = 0
+      @balance = initial_balance.to_i
+      @month = Time.now.mon
+    end
+
+    def is_at_transaction_limit
```

Comment:
The name of this method sets the expectation that it should return a boolean. So think about naming it something like `#check_transaction_limit`